test/intl402/NumberFormat/constructor-locales-arraylike.js

```diff
@@ -0,0 +1,23 @@
+// Copyright (C) 2018 Ujjwal Sharma. All rights reserved.
+// This code is governed by the BSD license found in the LICENSE file.
+
+/*---
+esid: sec-initializenumberformat
+description: >
+  Tests that the Intl.NumberFormat constructor accepts Array-like values for the
+  locales argument and treats them well.
+---*/
+
+const actual = Intl.NumberFormat({
+  length: 1,
+  1: 'en-US'
```

Comment:
I'm surprised that this is 1 rather than 0. I wonder if we're actually falling back to the system locale when running this test (so it might fail on some computers).

Reply:
@littledan this does seem to be a silly mistype. I'll make a PR fixing this ASAP.